ml/src/main/scala/ai/h2o/sparkling/ml/features/H2OTargetEncoder.scala

```diff
@@ -50,12 +50,14 @@ class H2OTargetEncoder(override val uid: String)
     val columnsToKeep = getInputCols() ++ Seq(getFoldCol(), getLabelCol()).map(Option(_)).flatten
     val ignoredColumns = dataset.columns.diff(columnsToKeep)
     val params = Map(
+      "data_leakage_handling" -> getHoldoutStrategy(),
       "blending" -> getBlendedAvgEnabled(),
-      "k" -> getBlendedAvgInflectionPoint(),
-      "f" -> getBlendedAvgSmoothing(),
+      "inflection_point" -> getBlendedAvgInflectionPoint(),
+      "smoothing" -> getBlendedAvgSmoothing(),
       "response_column" -> getLabelCol(),
       "fold_column" -> getFoldCol(),
       "ignored_columns" -> ignoredColumns,
+      "seed" -> getNoiseSeed(),
```

Comment:
doesn't SW expose the `noise` param?
If not, it means that noise is always enabled (not sure this is a good default, but this is the one that was chosen originally...), and the amount of noise is computed from the range of values in the target.
I think you should at least allow users to disable this noise: in other clients, it's done by setting `noise=0`, and they can otherwise specify that amount of noise, e.g. `noise=0.5` means that for each row the encoded value with be altered by some random noise comprised between -0.5 and +0.5.